### ISSUE TITLE
Discrepencies in Accuracy and Training time between versions 0.8.1 and 0.8.2

### ISSUE THREAD

**Issue by tiago1405** [jdb78/pytorch-forecasting#270]
- PyTorch-Forecasting version: 0.8.2
- PyTorch version: 1.7.1
- Python version: 3.7.8
- Operating System: Google Compute Engine AI Notebook Ubuntu 18.04

### Premise
**Accuracy**
Previously when running Pytorch Forecasting on version 0.8.1 I was getting a much higher accuracy than in  0.8.2. The MAE on version 0.8.1 for the same dataset using the same hyperparameters was consistently around 0.15 on average. While in 0.8.2 this has increased to around 0.5 (Figure below).
![InkedScreenshot_2021-01-18 Weights Biases_LI](https://user-images.githubusercontent.com/23642755/104959667-b61e2380-59a0-11eb-9cc3-fafae9663205.jpg)
In the above screenshot, the top values are runs using 0.8.2 (Purple, Red, Blue), and the bottom runs are using 0.8.1(Green, Dotted, Pink, Orange).

**Train Time**
Further, the training time between 0.8.1 and 0.8.2 has increased by 50% to 100% in some iterations (from 20-23 sec/it to 33-40 sec/it) on the same dataset using the same hyperparameters.
No part of my code, dataset, hyperparameters, or execution order was changed when running on 0.8.2 vs 0.8.1.

### Question
My question is the following: was there an issue with 0.8.1 that was giving me a better false result, or is this an issue with 0.8.2?

### Code to reproduce the problem
My code is essentially based on the example on the readthedocs page [(Here](https://pytorch-forecasting.readthedocs.io/en/latest/tutorials/stallion.html)) on how to implement the TFT model.

Please let me know if there's any further information that I may provide to help with this.

**Comment by jdb78** [comment]
There was an issue in 0.8.1 with incorrect re-scaling. I am only confused to see that it has actually gone worse for 0.8.2.
